marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.queries.tsx

```diff
@@ -379,6 +379,51 @@ describe('manage queries modal scenarios, developer role', () => {
         browsePage.waitForTableToLoad();
     });
 
+    // after(() => {
+    //     //clearing all the saved queries
+    //     cy.deleteSavedQueries();
+    // })
+
+    it('open manage queries, edit query', () => {
+        browsePage.getManageQueriesModalOpened();
```

Comment:
I think it would be better to combine these tests into a single test.

Reply:
I can combine them but I think shorter tests are more stable and much easier to debug.

Reply:
But isn't the whole point of e2e tests is to test real world user scenarios? Chaining user commands together is the best way to do that.

Reply:
combined.